lib/teamlab/response.rb

```diff
@@ -5,14 +5,26 @@ class Response
     def initialize(http_response)
       @code = http_response.code
       @success = @code < 400
+      err_msg = generate_err_msg(http_response) if @code >= 400
       if @success
         @body = http_response.to_hash
       else
-        fail TimeoutError, 'Portal is warming up' if http_response.parsed_response.include?('portal is being warmed')
-        fail "Error #{@code}" if @code >= 400
+        raise TimeoutError, 'Portal is warming up' if http_response.parsed_response.include?('portal is being warmed')
+        raise "Error #{@code}\n#{err_msg}" if @code >= 400
         @body = http_response.respond_to?(:parsed_response) && http_response.parsed_response.key?('result') ? http_response.parsed_response['result'] : http_response.to_hash
         @error = @body['error']['message'] if @body.key?('error') && @body['error'].key?('message')
       end
     end
+
+    def generate_err_msg(http_response)
+      err_msg = "API request failed\n\n"
```

Comment:
Better readability like this: https://github.com/ONLYOFFICE/testing-documentserver/blob/develop/data/test_data/input_output/test_data_chat.rb#L12
```
err_msg = "API request failed\n\n"\
          "original request:\n"\
...
```